### DISCUSSION TITLE
Undo termux-chroot command

### DISCUSSION THREAD

**Discussion by dextercva** [termux/termux-packages#14574] [Q&A] [answered]
I ran `termux-chroot` to check how it works. Now, it's classical Linux file system layout.
How should I go back to normal file system it was like before running `termux-chroot`?
Basically, the file system when we have since the installation of termux.

**Comment by sylirre** [comment, marked as answer]
Via CTRL-D key combination or `exit` command.

`termux-chroot` just starts a nested shell.

**Reply by dextercva** [reply]
Thanks. Is there any way to know nested shells running and its names?

**Reply by sylirre** [reply]
`echo $SHLVL`, basically a "shell level". Helps to identify how many nested shells are launched. Normally this is `1`, but after `termux-chroot` it will be `2`.

**Reply by sylirre** [reply]
Variable `$SHELL` will show the current shell program.

**Reply by dextercva** [reply]
Got it
